packages/api/src/graphql/article.ts

```diff
@@ -37,7 +38,8 @@ export const GraphQLArticleFilter = new GraphQLInputObjectType({
     published: {type: GraphQLBoolean},
     pending: {type: GraphQLBoolean},
     authors: {type: GraphQLList(GraphQLNonNull(GraphQLID))},
-    tags: {type: GraphQLList(GraphQLNonNull(GraphQLString))}
+    tags: {type: GraphQLList(GraphQLNonNull(GraphQLString))},
+    comments: {type: GraphQLList(GraphQLNonNull(GraphQLID))}
```

Comment:
There is no need to add comments to an article through the editor

Reply:
fixed